sulong/projects/com.oracle.truffle.llvm.nodes/src/com/oracle/truffle/llvm/nodes/intrinsics/multithreading/LLVMSemaphoreIntrinsics.java

```diff
@@ -0,0 +1,4 @@
+package com.oracle.truffle.llvm.nodes.intrinsics.multithreading;
+
+public class LLVMSemaphoreIntrinsics {
```

Comment:
unused?

Reply:
i was planning on implementing it too, but due to time reasons it's removed now.